google/cloud/bigquery/magics/magics.py

```diff
@@ -410,6 +437,15 @@ def _create_dataset_if_necessary(client, dataset_id):
         "Standard SQL if this argument is not used."
     ),
 )
+@magic_arguments.argument(
+    "--api_endpoint",
```

Comment:
We might actually want two API endpoints, one for the BigQuery client and one for the BigQuery Storage client.